packages/components/src/LightBox/LightBox.css

```diff
@@ -18,3 +18,61 @@
 .wrapper:global(.ril__outer .ril__toolbarItem) {
   color: var(--color-greyBlue--lighter);
 }
+
+.next,
+.prev {
+  display: flex;
+  position: absolute;
+  top: calc(50% - 20px);
+  z-index: 2;
+  width: 20px;
+  height: 34px;
+  padding: 30px 20px;
+  font-size: 18px;
+  font-weight: bold;
+  user-select: none;
+  background-color: rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+  justify-content: center;
+  align-items: center;
+}
+
+.next {
+  right: 0px;
+}
+
+.prev {
+  left: 0px;
+}
+
+.lightboxWrapper {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-overlay);
+  justify-content: space-between;
+  flex-direction: column;
+}
+
+.toolbar {
+  display: flex;
+  padding: var(--space-small) var(--space-base);
+  color: var(--color-greyBlue--lighter);
```

Comment:
Anywhere we use this text color let's replace with `var(--color-text--reverse--secondary)`